code/modules/mob/living/simple_animal/vore/otie.dm

```diff
@@ -84,6 +84,35 @@
 	glowyeyes = TRUE
 	eyetype = "photie"
 
+/mob/living/simple_animal/otie/redspace
+	name = "feral red otie"
+	desc = "Seems this ominous looking longdog has been infused with wicked infernal forces."
+	icon_state = "hotie"
+	icon_living = "hotie"
+	icon_dead = "hotie-dead"
+	icon_rest = "hotie_rest"
+	faction = "virgo3b"
```

Comment:
This should probably be changed but this isn't a deal breaker.

Maybe change the faction to "cult"? I think that's what faithless use. Is it?